src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -1847,5 +1847,39 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
       }
   }
 
-}
+  it should "return 404 when requesting bucket size for a non-existent workspace" in withTestWorkspacesApiServicesAndUser("reader-access") { services =>
+    Get(s"/workspaces/${testWorkspaces.workspace.namespace}/${testWorkspaces.workspace.name}x/bucketUsage") ~>
+      sealRoute(services.workspaceRoutes) ~>
+      check {
+        assertResult(StatusCodes.NotFound) { status }
+      }
+  }
+
+  // TODO: adjust user access for this test once the requirements of GAWB-1145 settle down
```

Comment:
There is some discussion about which types of users should see this information. No changes have made their way from the comment thread to the description yet, but there is talk of only showing this to owners or owners and writers.